drone/deploy/deploy.go

```diff
@@ -96,7 +96,7 @@ func deploy(c *cli.Context) error {
 
 	env := c.Args().Get(2)
 	if env == "" {
-		return fmt.Errorf("Please specify the target environment (ie production)")
+		return fmt.Errorf("Error: Please specify the target environment (e.g. production).")
```

Comment:
Nearly all messages are errors, so why do we highlight it here?

Reply:
I'm not sure, they are highlighted here: https://github.com/drone/drone-cli/blob/master/drone/internal/util.go#L31

Reply:
Can we agree on a certain format (either remove all `Error:`s or add it)?

Reply:
I would suggest to remove it